### ISSUE TITLE
Publish `TensorAuto/pi06-untrained` checkpoint (and pi0.7 follow-up)

### ISSUE THREAD

**Issue by shuheng-liu** [TensorAuto/OpenTau#239]
## Background

The `pi06` policy is implemented (`src/opentau/policies/pi06/`) and registered in `policies/factory.py`, and `configs/examples/pi06_training_config.json` exists, but there is no published initialization checkpoint. The README currently lists $\pi_{0.6}$ checkpoints as "Coming soon", which forces every new $\pi_{0.6}$ training run to assemble the backbone from scratch.

A separate $\pi_{0.7}$ checkpoint is tracked here as a follow-up — see the bottom of this issue.

## Ask

Publish a public, untrained $\pi_{0.6}$ initialization checkpoint to the Hub at:

```
TensorAuto/pi06-untrained
```

Composition:

- **VLM backbone**: load weights from [`google/gemma-3-4b-pt`](https://huggingface.co/google/gemma-3-4b-pt). The Gemma 3-4B PT checkpoint already bundles the SigLIP 400M vision tower, so we use it as-is to init the multimodal trunk inside `gemma3_with_expert.py`.
- **Action expert (~860M params)**: randomly initialized with the standard init used by the policy class — no pretrained weights.
- **Tokenizer / processor**: whatever ships with `google/gemma-3-4b-pt`.

Effectively this is a "from-scratch action expert on top of a frozen-init VLM" snapshot that any user can `from_pretrained("TensorAuto/pi06-untrained")` and start fine-tuning, instead of re-downloading + re-stitching the Gemma 3-4B + SigLIP 400M weights every time.

## Proposed implementation sketch (not part of this issue)

These are notes for whoever picks this up — happy to be overridden.

1. Add a small script under `src/opentau/scripts/` (e.g. `build_pi06_untrained.py`) that:
   - Instantiates `PI06Config` with the default 448×448, 5-step flow-matching settings.
   - Builds a `PI06Policy` via the factory, then loads `google/gemma-3-4b-pt` into the VLM submodule (vision tower + LM) — leaving the action expert at its default random init.
   - Calls `policy.save_pretrained(...)` and `tokenizer.save_pretrained(...)` to a local dir.
2. Push to `TensorAuto/pi06-untrained` (private → public after a sanity check).
3. Sanity-check by running `configs/examples/pi06_training_config.json` for a handful of steps starting from this checkpoint and confirming loss is finite and decreasing.
4. Update the README pretrained-checkpoints table: replace the "Coming soon" $\pi_{0.6}$ row with a link to `TensorAuto/pi06-untrained` and a one-line description.

## Acceptance criteria

- [x] `TensorAuto/pi06-untrained` exists on the Hub and is readable with correct API key.
- [x] `PI06Policy.from_pretrained("TensorAuto/pi06-untrained")` succeeds on a CPU box with no extra args.
- [x] A short smoke run (e.g. 40 steps of `pi06_training_config.json` initialized from the checkpoint) produces finite, decreasing loss.
- [x] README pretrained-checkpoints table updated.

## Follow-up: $\pi_{0.7}$ checkpoint

`pi07` (not to be confused with `pi07_paligemma`) is currently split into `high_level_planner` and `low_level` submodules. Once the $\pi_{0.7}$ policy lands as a stable, single user-facing entry (i.e. it's documented in the README and has an example config under `configs/examples/`), file/extend a follow-up to publish the analogous `TensorAuto/pi07-untrained` initialization checkpoint with the same recipe (pretrained VLM backbone, randomly initialized action expert).
